src/interface/ui/base/List.hpp

```diff
@@ -94,7 +94,7 @@ class List : virtual public ExternalInterface,
     {
         return qt_model_;
     }
-    auto shutdown(std::promise<void>& promise) noexcept -> void
+    auto xshutdown(std::promise<void>& promise) noexcept -> void
```

Comment:
what is xshutdown ?

Reply:
It is a proof that once the subclasses each have their own shut_down, no-one needs this shutdown() any more, well spotted.